### PR TITLE
feat(ui5-li): navigation mode is enabled

### PR DESCRIPTION
**Pull request by yanaminkova**

### CODE REVIEW

**Review by PetyaMarkovaBogdanova — DISMISSED** [review]

File: packages/main/src/types/ListItemType.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ const ListItemTypes = {
 	 * @type {Detail}
 	 */
 	Detail: "Detail",
+
+	/**
+	 * Enables the type of navigation, which is specified to add an arrow to the right side of the items and fires navigate-click event.
```
Comment (simonarangelova) [inline packages/main/src/types/ListItemType.js:30]:
```suggestion
	 * Enables the type of navigation, which is specified to add an arrow at the end of the items and fires navigate-click event.
```

File: packages/main/src/ListItem.js [hunk 2]
```diff
@@ -110,6 +110,10 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		navigated: {
```
Comment (ilhan007) [inline packages/main/src/ListItem.js:113]: if this is  a public property, it should be properly  documented

File: packages/main/src/ListItem.js [hunk 3]
```diff
@@ -364,6 +369,14 @@ class ListItem extends ListItemBase {
 		return this.type === ListItemType.Detail;
 	}
 
+	get typeNavigation() {
+		return this.type === ListItemType.Navigation;
+	}
+
+	get navigated() {
```
Comment (ilhan007) [inline packages/main/src/ListItem.js:376]: this getter is redundant, as "navigated" is part of the component metadata - there is already a getter with the same name

File: packages/main/src/ListItem.js [hunk 4]
```diff
@@ -214,13 +218,14 @@ class ListItem extends ListItemBase {
 	_onkeydown(event) {
 		super._onkeydown(event);
 
-		const itemActive = this.type === ListItemType.Active;
+		const itemActive = this.type === ListItemType.Active,
+			  navigationItem = this.type === ListItemType.Navigation;
```
Comment (ilhan007) [inline packages/main/src/ListItem.js:222]: itemNavigated would be more consistent with "itemActive" and you can reuse the new getter "typeNavigation", which checks the same:
this.type === ListItemType.Navigation;

**Review by ilhan007 — CHANGES_REQUESTED** [review]
@yanaminkova Thank you for the PR, there are few inline comments from my side

**Review by PetyaMarkovaBogdanova — APPROVED** [review]

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**